src/main/java/it/eng/idsa/businesslogic/processor/common/GetTokenFromDapsProcessor.java

```diff
@@ -56,16 +65,19 @@ public void process(Exchange exchange) throws Exception {
 			logger.debug("DAT token: {}", token);
 		} catch (Exception e) {
 			logger.error("Can not get the token from the DAPS server ", e);
+			publisher.publishEvent(new TrueConnectorEvent(TrueConnectorEventType.CONNECTOR_TOKEN_FETCH_FAILURE, multipartMessage));
 			rejectionMessageService.sendRejectionMessage((Message) exchange.getProperty("Original-Message-Header"), RejectionReason.NOT_AUTHENTICATED);
 		}
 
 		if (token == null) {
```

Comment:
Can we change this not null and not empty to IsBlank from StringUtils?

Reply:
For now, we will leave this one as is and address it in separate issue/improvement